src/Sylius/Bundle/OrderBundle/Resources/config/doctrine/model/OrderItem.orm.xml

```diff
@@ -35,6 +35,9 @@
 
         <many-to-one field="order" target-entity="Sylius\Component\Order\Model\OrderInterface" inversed-by="items">
             <join-column name="order_id" referenced-column-name="id" nullable="false" on-delete="CASCADE" />
+            <cascade>
+                <cascade-persist />
+            </cascade>
```

Comment:
This seems terribly wrong, cart item should not cascade persist cart itself, it's the other way around. :/